src/main/scala/com.snowplowanalytics.s3/loader/KinesisSourceExecutor.scala

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (c) 2014-2015 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.s3.loader
+
+// Logging
+import org.slf4j.LoggerFactory
+
+// AWS Kinesis Connector libs
+import com.amazonaws.services.kinesis.connectors.{
+  KinesisConnectorConfiguration,
+  KinesisConnectorExecutorBase,
+  KinesisConnectorRecordProcessorFactory
+}
+
+// AWS Client Library
+import com.amazonaws.services.kinesis.clientlibrary.lib.worker.KinesisClientLibConfiguration
+import com.amazonaws.services.kinesis.clientlibrary.lib.worker.Worker
+import com.amazonaws.services.kinesis.metrics.interfaces.IMetricsFactory
+
+// Java
+import java.util.Date
+
+// Tracker
+import com.snowplowanalytics.snowplow.scalatracker.Tracker
+
+// This project
+import sinks._
+import serializers._
+import model._
+
+/**
+ * Boilerplate class for Kinessis Conenector
+ */
+class KinesisSourceExecutor(
+  config: KinesisConnectorConfiguration, 
+  s3LoaderConfig: S3LoaderConfig,
```

Comment:
it seems to me you don't need the whole S3LoaderConfig, just the initial position and initial timestamp, same as the es-loader